Themes/default/languages/index.english.php

```diff
@@ -150,7 +150,7 @@
 $txt['notify'] = 'Notify';
 $txt['unnotify'] = 'Unnotify';
 // Use numeric entities in the below string.
-$txt['regards_team'] = "Regards,\nThe " . $context['forum_name'] . ' Team.';
+$txt['regards_team'] = 'Regards,' . "\n" . 'The %1$s Team.';
```

Comment:
I don't understand enough about PHP (or the SMF requirements) to address the newline.

Reply:
Do the double quotes mess up the `%1$s` part? If not, `$txt['regards_team'] = "Regards,\nThe %1$s Team.";` should work.

Reply:
I thought so, but apparently [it should work like you suggest](https://www.php.net/manual/en/language.types.string.php#language.types.string.parsing).

Reply:
Hmm, but then again, this creates loads of `8: Undefined variable: s` errors in my test install… :cry:

Reply:
Just committed a potential fix (6fdc40e8f), which I'll test tomorrow…